components/ui/radio-group.vue

```diff
@@ -63,16 +63,27 @@
         this.$emit('input', item);
       },
       style(item) {
-        if(!item.color) {
-          return '';
+        if(!item.color) return '';
+
+        // Parse HEX colors
+        let outline = '';
+        let match = item.color.match(/#([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})/);
```

Comment:
Probably need to match lowercase too, or match with `item.color.toUpperCase()` instead

Reply:
That's right